packages/fungible-token/bridge-fungible-token/src/main.sw

```diff
@@ -117,36 +125,46 @@ impl MessageReceiver for Contract {
 
 impl FungibleBridge for Contract {
     #[storage(read, write)]
-    fn claim_refund(originator: b256, asset: b256) {
-        let stored_amount = storage.refund_amounts.get(originator).get(asset).read();
-        require(stored_amount != ZERO_B256, BridgeFungibleTokenError::NoRefundAvailable);
+    fn claim_refund(from: b256, token_address: b256, token_id: b256) {
+        let asset = sha256((token_address, token_id));
+        let amount = storage.refund_amounts.get(from).get(asset).read();
```

Comment:
```suggestion
        let amount = storage.refund_amounts.get(from).get(asset).try_read().unwrap_or(ZERO_B256);
```
If no value has been inserted to the key `asset`, then reading will return `None`. The suggestion is only necessary when we're trying to read from a key that may not yet have had a value inserted, in which case we should always use `.try_read()`. 
This applies to other instances in this file too, where `read` should be replaced with `try_read` when using `get` on `StorageMap`

Reply:
Done